src/neo/Consensus/ConsensusService.cs

```diff
@@ -185,8 +187,42 @@ private void InitializeConsensus(byte viewNumber)
             {
                 ChangeTimer(TimeSpan.FromMilliseconds(Blockchain.MillisecondsPerBlock << (viewNumber + 1)));
             }
+
+            // Try to speed up consensus if cached future payloads exists
+            if (context.HasFuturePayloads())
+            {
+                TryToConsumeFuturePayload(context.FuturePreparationPayloads);
+                if (context.HasFuturePayloads())
+                    TryToConsumeFuturePayload(context.FutureCommitPayloads);
+                if (context.HasFuturePayloads())
+                    TryToConsumeFuturePayload(context.FutureChangeViewPayloads);
+                if (context.HasFuturePayloads())
+                    TryToConsumeFuturePayload(context.FutureRecoveryPayloads);
+            }
```

Comment:
I think we should consume the `ChangeView` payloads first.

Reply:
That is true, such as in the Recovery.

Reply:
Done.